app/code/Magento/Customer/Controller/Account/CreatePost.php

```diff
@@ -119,6 +119,11 @@ class CreatePost extends AbstractAccount implements CsrfAwareActionInterface, Ht
      */
     protected $session;
 
+    /**
+     * @var ScopeConfigInterface
+     */
+    protected $scopeConfig;
```

Comment:
Could you explain why you added this property?

Reply:
@gokulra92, the $scopeConfig is already declared below as `private $scopeConfig`.

Reply:
Yeah, please remove this line